### DISCUSSION TITLE
Is Plausible activly developed? Roadmap?

### DISCUSSION THREAD

**Discussion by MickL** [plausible/analytics#4499] [Anything Else]
Hello, I really like the UI and DX of Plausible. It integrates well with Nuxt and the UI is very nice and responsive. To switch over from etracker we are missing super important features. I saw this features are highly requested in the discussions and in the [feature request tracker](https://feedback.plausible.io/). All of this features are part of etracker and Matomo but not Plausible:

1. [User journeys / navigation flows](https://feedback.plausible.io/178) per user-session
Click a session and she where he came from (Google Ads campagne name), each page he navigated, how much time he spent on each page and what custom events he triggered in his journy

2. [User journeys / navigation flows](https://feedback.plausible.io/178) overall
See a visual flow diagram what paths users take and where they exit the page

3. [Scroll depth](https://feedback.plausible.io/183)
For each page, what percentage scrolled to what part? 

4. [Heat maps](https://feedback.plausible.io/308)
For each page, where did users click?

5. [Export conversions to Google Ads](https://feedback.plausible.io/210)
This happens by file upload in etracker

6. [Multilingual dashboard](https://feedback.plausible.io/186)
Not everyone speaks english, especially in latin-speaking countries like France, Spain, Portugal and Italy.

My biggest question is: Is Plausible actively developed? Are this features in development? Is there a roadmap?

**Comment by metmarkosaric** [comment]
thanks! [User journeys / navigation flows](https://plausible.io/docs/user-journeys) and [scroll depth tracking](https://plausible.io/docs/scroll-depth) are available in Plausible at the time of my reply. the product is actively developed. you can see our roadmap here: https://feedback.plausible.io/roadmap. that's the same place where you can also see feature request from our users (and where you can add new ideas). we keep all of the feature requests on our feedback board where we also share updates on what we're working on next
